docs/csharp/package-management.md

```diff
@@ -65,20 +75,22 @@ In C# Dev Kit, you can perform NuGet package operations in either of two ways: b
 
 3. Next, choose which package you want to remove from the list of packages that are currently installed in your project.
 
-4. C# Dev Kit then performs the necessary changes and updates your project file and references.
+4. The C# Dev Kit then performs the necessary changes and updates your project file and references.
 
 ## Dependency management
 
 ### Automatic NuGet restore
 
-For a .NET project, a package restore happens automatically when you create a project from a template, build, load, or make changes to an SDK-style project. You can view in the progress and logs in the Ouput panel.
+For a .NET project, a package restore happens automatically when you create a project from a template, build, load, or when making changes to an SDK-style project. You can view in the progress and logs in the Ouput panel.
 
 ![Automatic NuGet package restore](images/package-management/automatic-nuget-package-restore.png)
 
 For projects that use `<PackageReference>`, you can see the package references in the **Solution Explorer** section of Visual Studio Code.
 
 ![Package references in the Solution Explorer](images/package-management/package-references-solution-explorer.png)
 
-Packages that don't install properly when a restore happens or you run a build will show error icons in **Solution Explorer**.
+Packages that don't install properly when a restore happens or when you run a build, will show error icons in **Solution Explorer**.
+
+**Note**: At this time, you cannot right-click on projects to manage your NuGet packages and there is not a NuGet Package Manager user interface in Visual Studio Code. 
 
-**Note**: At this time, you cannot right-click on projects to manage your NuGet packages and there is not a NuGet Package Manager user interface in Visual Studio Code. For more information on managing packages, see [Install and manage NuGet packages with the dotnet CLI](https://learn.microsoft.com/nuget/consume-packages/install-use-packages-dotnet-cli).
+For more information on managing packages, see [Install and manage NuGet packages with the dotnet CLI](https://learn.microsoft.com/nuget/consume-packages/install-use-packages-dotnet-cli).
```

Comment:
The link to docs is related to the note and should be included in the blockquote. Please revert this change.

Reply:
ok, will do.

Reply:
Done.